specification/src/main/asciidoc/jakarta-concurrency.adoc

```diff
@@ -1034,14 +967,14 @@ each task provider’s implementation.
 [arabic]
 . A task instance that starts a transaction must complete the
 transaction before starting a new transaction.
-. The task provider uses the javax.transaction.UserTransaction interface
+. The task provider uses the `jakarta.transaction.UserTransaction` interface
 to demarcate transactions.
-. Transactions are demarcated using the begin(), commit() and rollback()
-methods of the UserTransaction interface.
+. Transactions are demarcated using the `begin()`, `commit()` and `rollback()`
+methods of the `UserTransaction` interface.
 . While an instance is in an active transaction, resource-specific
 transaction demarcation APIs must not be used (e.g., if a
-javax.sql.Connection is enlisted in the transaction instance, the
-Connection.commit() and Connection.rollback() methods must not be used).
+`javax.sql.Connection` is enlisted in the transaction instance, the
```

Comment:
This error was already present in the old spec, but javax.sql.Connection has the wrong package name. This ought to be java.sql.Connection